### PR TITLE
issue: #48: Asksusi.com version 3

### PR DESCRIPTION
**Pull request by sanchittechnogeek**
Finally ```Asksusi.com``` version: ```v3``` is here: 

Issue: #48

> Design inspiration: [Google Allo](https://allo.google.com/) , [Google Assistant](https://assistant.google.com/)

# Change log:
### Features:
- All new design (Flat design)
- Now using angular includes
- New animated loading bar
- Beautifully indented code for easy reading
- Optimized images for faster loading
- Using custom anchor smooth scroll

### Parts that still need improvement:
- susi logo
- adding required links to fb page, twitter page, app store and play store
- Adding reviews from more people
- Improving the directory structure.

### Known issues:
- The page will not load the  templates from ng-view route or chat route as it expects the URL to be relative to the root of the domain. (This issue may or may not occur when using custom domain). Quick fix for this issue: add ```<base href="/repo_name/" />``` to the head of the html page.
 
### Next steps:
- Experimenting with rendering /chat route in the landing page itself as a virtual device , so that the user do not have to jump to another page to see the demo / susi in action. more info soon!
- Experimenting with Material Design
- New susi fluid like animated logo
- Adding an angualr based contact form
- Adding a subscribe to mailing list option

#### Instructions for adding your reviews to the reviews section.

The reviews section consists of two parts:
The structure is like this :-
- Active indicators (these are the small dots below the review message)
```sh
<!-- Bottom Carousel Indicators -->
<ol class="carousel-indicators">
    <!-- Quote 1: -->
    <li data-target="#quote-carousel" data-slide-to="0" class="active"></li>
    <!-- Quote 2:-->
    <li data-target="#quote-carousel" data-slide-to="1"></li>
    <!-- Quote 3:-->
    <li data-target="#quote-carousel" data-slide-to="2"></li>
</ol>
```
- Your Review message
```sh
<!-- Carousel Slides / Quotes -->
<div class="carousel-inner">

<!-- Quote 1 -->
<div class="item active">
<div class="row">
<div class="col-sm-12">
<p>&ldquo;Susi is great! , it is able to understand and reply what user says. I have also encountered some bugs in the beta stage which needs to be fixed to make it better. :) &rdquo;</p>
<small><strong>Sanchit Mishra</strong></small>
</div>
</div>
</div>
<!-- END-Quote 1: -->

<!-- Quote 2: -->
<div class="item">
<div class="row">
<div class="col-sm-12">
<p>&ldquo; Your Review &rdquo;</p>
<small><strong>student name</strong></small>
 </div>
</div>
</div>
<!-- END-Quote 2: -->

<!-- Quote 3: -->
<div class="item">
<div class="row">
<div class="col-sm-12">
<p>&ldquo; Your Review &rdquo;</p>
<small><strong>student name</strong></small>
</div>
</div>
</div>
<!-- END-Quote 3: -->

</div>
<!-- END: Carousel Slides / Quotes -->
```

1. First uncomment (first 3 are already added as comments) or add this line
`<li data-target="#quote-carousel" data-slide-to="1"></li>` provided below inside the ol tags for adding active indicators. You have to also add a no. into `data-slide-to=""` attribute to tell the JS to which message it needs to slide to.   

2. Now you have to uncomment (first 3 are already added as comments) or add this:
```sh
<!-- Quote No. : -->
<div class="item">
<div class="row">
<div class="col-sm-12">
<p>&ldquo; Your Review &rdquo;</p>
<small><strong>student name</strong></small>
</div>
</div>
</div>
<!-- END-Quote No. : -->
```
just after the end quote comments of the previous reviews. After adding it replace `Your Review` inside the `<p></p> tags` with your review message. Also do mention the quote no. in the quote No. / End Quote No. comment lines 

**Note**: The no. of active indicators should be equal to the no. of review messages.

### What i have used for new design:
- Bootstrap framework
- http://coderthemes.com/velonic_wb_3/frontend/ : Helped for the base of the flat design
- Pace.js : Animated loading bars
- Custom made temporary susi animated logo SVG
- Google Fonts: Indie Flower
- Az screen recorder for recording installation tutorials
- https://github.com/ellerbrock/open-source-badges : Open source badge

Live link with New Improved Design: [Click here!!!](https://sanchittechnogeek.github.io/asksusi.com/)

asksusi

### CODE REVIEW

**Comment by sanchittechnogeek** [comment]
@mariobehling @niccokunzmann @Princu7 @jig08 @hpdang please try out the new design. :)

**Comment by mariobehling** [comment]
Very nice. On first glance could you make some changes, please:
* Keep the top bar white or light grey and keep it there, when the user scrolls
* Make the first screen white, if possible
* Take out references to iOS (We dont have it)
* provide link to the original theme template

**Comment by sanchittechnogeek** [comment]
sure.

**Comment by jig08** [comment]
@sanchittechnogeek Impressive work :)

**Comment by robbyoconnor** [comment]
@fossasia/asksusi, When this is merged, [approve his task](https://codein.withgoogle.com/dashboard/task-instances/6024888529715200/)

**Comment by sudheesh001** [comment]
This looks great. As suggested, can you please remove that logo that appears as soon as the landing page opens and also in the favicon.

One more suggestion being
-[ ]  Remove placeholder `Lorem ipsum` text in the Reviews/Testimonials sections.

Looks good to me with the changes. Great work @sanchittechnogeek

**Comment by sanchittechnogeek** [comment]
Done.

**Comment by jig08** [comment]
Ah, just noticed
@sanchittechnogeek Could you remove that unknown blue favicon as well ?

**Comment by sanchittechnogeek** [comment]
@jig08 I have already removed it , please refresh your browser cache :)

**Comment by mariobehling** [comment]
@sanchittechnogeek There is no download link to the susi android app. For the moment, please provide a link to the app in the repository. In the former version there was an app link. Please use this one.

**Comment by mariobehling** [comment]
Please also add the susi logo. All artwork is here: https://github.com/fossasia/susi_artwork

File: index.html [hunk 1]
```diff
@@ -1,78 +1,84 @@
...
+<body ng-controller="mainController">
+    <div ng-view></div>
+    <ng-include src="'./includes/footer.html'"></ng-include>
+    <!-- End Footer -->
+    <!-- js placed at the end of the document so the pages load faster -->
+    <script src="./assets/js/jquery.min.js"></script>
+    <script src="./assets/js/bootstrap.min.js"></script>
+    <script src="./assets/js/pace.min.js"></script>
+    <script src="./assets/js/wow.min.js"></script>
+    <!-- Jquery easing , SmoothScroll
```
Comment (jayvdb) [inline index.html]: Why is this commented out?
Sorry if I am ignorant about something here.

**Comment by sanchittechnogeek** [comment]
@mariobehling I have added the download links to the app , if you click the github logo in the downloads section or 'here' text in the how it works section, susi-debug.apk will be downloaded

File: index.html (hunk 1 above)
Comment (sanchittechnogeek) [inline index.html]: I was testing the the jQuery , but it is not working well with angular js , so I have commented it out as I will experiment with it later, instead I have added angular directives for this :)

**Comment by mariobehling** [comment]
@sanchittechnogeek Thanks. Please add a download link right on the start screen below the test. Cheers.

**Comment by mariobehling** [comment]
Also, in the place where the former logo used to be, please add the Susi logo.

**Comment by sanchittechnogeek** [comment]
Sure , 😃

File: index.html (hunk 1 above)
Comment (jayvdb) [inline index.html]: So does that mean we dont need these files that you added, like `assets/js/jquery-ui-1.10.1.custom.min.js` and `assets/js/jquery.easing.1.3.min.js` ?

Comment (sanchittechnogeek) [inline index.html]: No, not for now. I will remove them in the next commit.

**Pull request merged**